pkg/config/config_test.go

```diff
@@ -32,3 +32,25 @@ func TestUnknownConfigFields(t *testing.T) {
 		require.Contains(t, err.Error(), "field UnknownConfigurationField not found in type config.Config")
 	})
 }
+
+func TestLoadFileWithMissingDefaultConfigPath(t *testing.T) {
+	originalPath := DefaultConfigPath
+	temporaryPath := filepath.Join(filepath.Dir(originalPath), "missing-config")
+
+	if _, err := os.Stat(originalPath); os.IsNotExist(err) {
+		err = os.MkdirAll(originalPath, os.ModePerm)
+		require.NoError(t, err)
+	}
+
+	t.Cleanup(func() {
+		err := os.Rename(temporaryPath, originalPath)
+		require.NoError(t, err)
+	})
+
+	err := os.Rename(originalPath, temporaryPath)
+	require.NoError(t, err)
+
+	_, err = LoadFile(filepath.Join(DefaultConfigPath, "protocol.privnet.yml"))
+	require.Error(t, err)
+	require.Contains(t, err.Error(), "doesn't exist")
```

Comment:
Why error? It must load the embedded config, because the default config-path is used with a known config file.